### PR TITLE
missing italian translation fixed

### PR DESCRIPTION
**Pull request by vicvega83**
`` system/manager/LanguageManager/strings.json``
was missing the italian dialogues

### CODE REVIEW

File: system/manager/LanguageManager/strings.json [hunk 1]
```diff
@@ -19,6 +22,9 @@
 		],
 		"de": [
 			" und auch "
+		],
+		"it": [
+			"ed anche"
```
Comment (Psychokiller1888) [inline system/manager/LanguageManager/strings.json]: needing the spaces here, before and after

File: system/manager/LanguageManager/strings.json [hunk 2]
```diff
@@ -39,6 +45,14 @@
...
 			"in Raum ",
 			"in "
+		],
+		"it": [
+			"nella",
+			"nello",
+			"nel",
+			"nei",
+			"dentro",
+			"in",
```
Comment (Psychokiller1888) [inline system/manager/LanguageManager/strings.json]: Needs white space at then end

Comment (Psychokiller1888) [reply system/manager/LanguageManager/strings.json]: I meant at the end of the words :)

Comment (vicvega83) [reply system/manager/LanguageManager/strings.json]: yes :) corrected

**Pull request merged**